app/layout.tsx

```diff
@@ -14,9 +16,9 @@ export default function RootLayout({
   return (
     <html lang="en">
       <body className="w-full flex flex-col items-center justify-between h-screen">
-        {children}
+        <Provider>{children}</Provider>
         {/* change the next line from true instead of children later */}
-        {true && (
+        {false && (
```

Comment:
My this is tricky. We only want the nav bar on logged in components if you just replace it here with false, won't have it anymore for the logged in components. We need to create the logic for this because either main or your branch is not really reflecting the changes. 
Would you mind doing it or should I create a new ticket and find someone else?

Reply:
Or, we don't really have any pages that display something in the logged in state as of right now, do we?

Reply:
I found a way! will update this and push again